website/docs/r/external_network_v2.html.markdown

```diff
@@ -0,0 +1,176 @@
+    }
+  }
+
+  ip_scope {
+    # enabled       = true # by default
+    gateway       = "14.14.14.1"
+    prefix_length = "24"
+
+    static_ip_pool {
+      start_address = "14.14.14.10"
+      end_address   = "14.14.14.15"
+    }
+    
+    static_ip_pool {
+      start_address = "14.14.14.20"
+      end_address   = "14.14.14.25"
+    }
+  }
+}
+```
+
+## Example Usage (NSX-V backed external network)
+```hcl
+data "vcd_vcenter" "vc" {
+  name = "vc1"
+}
+
+data "vcd_portgroup" "sw" {
+  name = "TestbedPG"
+  type = "DV_PORTGROUP"
+}
+
+resource "vcd_external_network_v2" "ext-net-nsxv" {
+  name        = "TestAccVcdExternalNetworkBasic"
+  description = "Test External Network"
+
+  vsphere_network {
+    vcenter_id     = data.vcd_vcenter.vc.id
+    portgroup_id   = data.vcd_portgroup.sw.id
+  }
+
+  ip_scope {
+    gateway      = "192.168.30.49"
+    netmask      = "24"
+    dns1         = "192.168.0.164"
+    dns2         = "192.168.0.196"
+    dns_suffix   = "company.biz"
+
+    static_ip_pool {
+      start_address = "192.168.30.51"
+      end_address   = "192.168.30.62"
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A unique name for the network
+* `description` - (Optional) Network friendly description
+* `ip_scope` - (Required) One or more IP scopes for the network. See [IP Scope](#ipscope) below for details.
+* `vsphere_network` - (Optional) One or more blocks of [vSphere Network](#vspherenetwork)..
+* `nsxt_network` - (Optional) NSX-T network definition. See [NSX-T Network](#nsxtnetwork) below for details.
+
+<a id="ipscope"></a>
+## IP Scope
+
+* `gateway` - (Required) Gateway of the network.
+* `prefix_length` - (Required) Network prefix.
+* `static_ip_pool` - (Required) IP ranges used for static pool allocation in the network.  See [IP Pool](#ip-pool) below for details.
+* `dns1` - (Optional) Primary DNS server. **Only valid** for NSX-V networks.
+* `dns2` - (Optional) Secondary DNS server. **Only valid** for NSX-V networks.
+* `dns_suffix` (Optional) A FQDN for the virtual machines on this network. **Only valid** for NSX-V networks.
+* `enabled` - (Optional) Default is `true`.
+
+<a id="ip-pool"></a>
+## IP Pool
+
+* `start_address` - (Required) Start address of the IP range
+* `end_address` - (Required) End address of the IP range
+
+<a id="vspherenetwork"></a>
+## vSphere Network
+
+* `vcenter_id` - (Required) vCenter ID. Can be looked up using [`vcd_vcenter`](/docs/providers/vcd/d/vcenter.html) data source.
+* `portgroup_id` - (Required) vSphere portgroup ID. Can be looked up using  [`vcd_portgroup`](/docs/providers/vcd/d/portgroup.html) data source.
+
+<a id="nsxtnetwork"></a>
+## NSX-T Network
+
+* `nsxt_manager_id` - (Required) NSX-T manager ID. Can be looked up using [`vcd_nsxt_manager`](/docs/providers/vcd/d/nsxt_manager.html) data source.
+* `nsxt_tier0_router_id` - (Required) NSX-T Tier-0 router ID. Can be looked up using
```

Comment:
In contrast to this "normal" Tier-0 router, where/when Terraform user should use the VRF Tier-0 router?

Reply:
In fact it can be supplied here as well and the datasource does look it up.